userguide/jails.rst

```diff
@@ -488,6 +489,16 @@ of these options.
    +--------------------------+--------------+---------------------------------------------------------------------------------------------------------------------+
 
 
+.. warning::
+   :name: security warning
+
+   :guilabel:`allow_raw_sockets` enables Inernet Control Message
+   Protocol (ICMP) and
```

Comment:
It doesn't strictly allow tcpdump, but the use of tcpdump or any other tools that require raw sockets, like nmap.